pennylane/devices/default_qutrit_mixed.py

```diff
@@ -85,10 +88,65 @@ def accepted_sample_measurement(m: qml.measurements.MeasurementProcess) -> bool:
     return isinstance(m, qml.measurements.SampleMeasurement)
 
 
+@qml.transform
+def warn_measurement_error_state(
+    tape: qml.tape.QuantumTape,
+) -> tuple[Sequence[qml.tape.QuantumTape], Callable]:
+    """Warns if a measurement is an analytic State or density_matrix measurement that measurement
+    error is not applied.
+
+    Args:
+        tape (QuantumTape, .QNode, Callable): a quantum circuit.
+
+    Returns:
+        qnode (pennylane.QNode) or quantum function (callable) or tuple[List[.QuantumTape], function]:
+
+        The unaltered input circuit. The output type is explained in :func:`qml.transform <pennylane.transform>`.
+    """
+    if not tape.shots:
+        for m in tape.measurements:
+            if isinstance(m, qml.measurements.StateMP):
+                warnings.warn(f"Measurement {m} is not affected by measurement error.")
+
+    return (tape,), null_postprocessing
+
+
+# PR-comment: separate function so that you can use this to redefine the measurment error without needing new function
+def get_measurement_errors(damping_measurement_gammas, trit_flip_measurement_probs):
+    r"""Get the list of measurement errors that should be applied to each
+
+    Args:
+        damping_measurement_gammas (List[float]): Gamma inputs for qml.QutritAmplitudeDamping channel
```

Comment:
Should I add a citation to why this is added?

Reply:
Perhaps not in the args part of the docstring, but in the description above ^

Reply:
I made another details section and added it there.